fluss-flink/fluss-flink-2.0/src/main/java/org/apache/flink/api/connector/sink2/Sink.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.api.connector.sink2;
+
+import org.apache.flink.api.common.operators.MailboxExecutor;
+import org.apache.flink.metrics.groups.SinkWriterMetricGroup;
+
+import java.io.IOException;
+import java.io.Serializable;
+
+/**
+ * Placeholder class to resolve compatibility issues.(Providing
+ * org.apache.flink.api.connector.sink2.Sink.InitContext).
```

Comment:
Add a comment that this placeholder class can be removed once we drop the support of Flink v1.18 which requires the legacy `InitContext` interface.